src/CMakeLists.txt

```diff
@@ -0,0 +1,46 @@
+# libcps
+add_library(libcps cps/env.cpp cps/loader.cpp cps/printer.cpp cps/search.cpp
+                   cps/utils.cpp cps/version.cpp)
+
+set(CMAKE_CXX_STANDARD 23)
```

Comment:
This is different than Meson, which has C++17 (I picked C++17 because I wanted to have broader compatibility), but whatever we are using we should use the same standard for all build systems unless there's a really compelling reason not to.

Reply:
Ooopsie. That should have never been there. I was testing whether `std::unreachable` detection is working as expected.